packages/compass-collection/src/stores/tabs.ts

```diff
@@ -206,25 +210,25 @@ const doCloseTab = (state, action) => {
   }, []);
 };
 
-const doCollectionDropped = (state, action) => {
-  const tabs = state.filter((tab) => {
+const doCollectionDropped = (state: any, action: any) => {
+  const tabs = state.filter((tab: any) => {
     return tab.namespace !== action.namespace;
   });
   if (tabs.length > 0) {
-    if (tabs.findIndex(tab => tab.isActive) < 0) {
+    if (tabs.findIndex((tab: any) => tab.isActive) < 0) {
       tabs[0].isActive = true;
     }
   }
   return tabs;
 };
 
-const doDatabaseDropped = (state, action) => {
-  const tabs = state.filter((tab) => {
+const doDatabaseDropped = (state: any, action: any) => {
+  const tabs = state.filter((tab: any) => {
     const tabDbName = toNS(tab.namespace).database;
     return tabDbName !== action.name;
   });
   if (tabs.length > 0) {
-    if (tabs.findIndex(tab => tab.isActive) < 0) {
+    if (tabs.findIndex((tab: any) => tab.isActive) < 0) {
```

Comment:
Totally up to you, but `tab: any` or `tabs: any[]` pops up a lot here, maybe it’s worth actually typing them?